src/MeoAsstGui/Resources/Localizations/ja-jp.xaml

```diff
@@ -206,6 +206,8 @@
     <system:String x:Key="Quantity">ドロップ数</system:String>
     <system:String x:Key="StageSelect">ステージ</system:String>
     <system:String x:Key="StageSelect2">代替ステージ</system:String>
+    <system:String x:Key="RemainingSanityStage">余剰理性の消費方法</system:String>
```

Comment:
maybe too long

Reply:
just use 余剰理性, and use a popup tip to show "余剰理性の消費方法"?

Reply:
EN: Remining Sanity / 15 (Byte 7+Space)
JP:余剰理性の消費方法 / 9 (2Byte 18)

Plan
JP:余剰理性消費方法 / 8 (2Byte 16)
JP:余剰理性の消費 / 7 (2Byte 14)

"方法"を削っても表示的には問題ないと思います。

I think it would be fine to remove the word "方法".